galsim/base.py

```diff
@@ -147,12 +147,6 @@ def __str__(self):
 
 _galsim.SBGaussian.__getinitargs__ = lambda self: (
         self.getSigma(), self.getFlux(), self.getGSParams())
-# SBProfile defines __getstate__ and __setstate__.  We don't actually want to use those here.
```

Comment:
Mostly it's that we don't have anything that returns a bare SBProfile to python.  So we always know what actual type it is, and we don't need the generic prescription for pickling an SBProfile object.  The specific ones (e.g. `_galsim.SBGaussian.__getinitargs__`) are sufficient.

Aside: In the next #809 PR, I'll go another step and remove even these, since I won't depend on the various SBProfile types being picklable to make the python type picklable.  In fact, it will even be option whether a specific GSObject subclass has any `_sbp` attribute at all or whether the implementation is defined in pure python.